### PR TITLE
motors: Document battery compensated thrust

### PR DESCRIPTION
**Pull request by jonasdn**
Data for this comes from:
  https://wiki.bitcraze.io/misc:investigations:thrust

### CODE REVIEW

File: src/drivers/src/motors.c [hunk 1]
```diff
@@ -231,10 +231,53 @@ void motorsSetRatio(uint32_t id, uint16_t ithrust)
...
+    // => p = -0.00062390   0.08835522   0.06865956
+    //
+    // And to get the PWM as a percentage we would need to divide the
+    // Voltage needed with the Supply voltage.
+    //
     if (motorMap[id]->drvType == BRUSHED)
     {
       float thrust = ((float)ithrust / 65536.0f) * 60;
-      float volts = -0.0006239f * thrust * thrust + 0.088f * thrust;
+      float volts = -0.0006239f * thrust * thrust + 0.088f * thrust + 0.06866;
```
Comment (jonasdn) [inline src/drivers/src/motors.c:280]: Was the constant left out for a reason? Is it negligible?

Comment (tobbeanton) [reply src/drivers/src/motors.c]: Yes the constant was left out as we need zero thrust to output zero.

**Comment by tobbeanton** [comment]
Nice clarifying comment. What about breaking it out to its own function as well?

**Comment by jonasdn** [comment]
> Nice clarifying comment. What about breaking it out to its own function as well?

I tried that and force-pushed, what do you think?

**Review by krichardsson — COMMENTED** [review]
Great to document this stuff and move missing info from the wiki! 

Would it be better to put it in the repo docs instead? This is also only valid for the "normal" Crazyflie, how do we handle the Bolt and BigQuad decks? I suppose we will add more info about those when we finalize them.

**Pull request merged**